src/vs/workbench/services/workspaces/common/workspaceTrust.ts

```diff
@@ -528,6 +548,11 @@ export class WorkspaceTrustManagementService extends Disposable implements IWork
 			return { trusted: true, uri };
 		}
 
+		// Uri is trusted automatically by the remote
```

Comment:
Are we missing the `this.isTrustedVirtualResource()` here?

Reply:
this will be required if file system provider's can provide canonical uris, but they cannot at the current time